metabase/metabase
# Move question lineage to AppBar

Epic https://github.com/metabase/metabase/pull/23569

How to test:
- Open a saved question and close the sidebar
- You should see the collection breadcrumbs in the app bar
- Change the question without saving it. You should see `Started from ...` lineage
- Click on the link from the lineage. Make sure it navigates to the correct question 

<img width="1728" alt="Screenshot 2022-06-27 at 21 21 45" src="https://user-images.githubusercontent.com/8542534/176010806-59764281-efff-434d-aa0f-29a6aea10d81.png">
